sound/soc/sof/ipc3-topology.c

```diff
@@ -2241,7 +2241,7 @@ static int sof_tear_down_left_over_pipelines(struct snd_sof_dev *sdev)
 		for_each_pcm_streams(dir) {
 			struct snd_pcm_substream *substream = spcm->stream[dir].substream;
 
-			if (!substream || !substream->runtime)
+			if (!substream || !substream->runtime || spcm->stream[dir].suspend_ignored)
```

Comment:
commit message: widgets are set up when a PCM he_params

did you mean with? during?